### ISSUE TITLE
LibrarySettings.logLevel doesn't appear to do anything

### ISSUE THREAD

**Issue by iankerr** [Tealium/tealium-kotlin#131]
Maybe I'm mistaken as to the purpose of what the logLevel property is for in LibrarySettings. I thought it might be used to override the logging level set by using the Environment enum in TeliumConfig.

But this is the only time I'm seeing the logLevel of the Logger being set. Telium.kt, line 139:

```
        Logger.logLevel = when (config.environment) {
            Environment.DEV -> LogLevel.DEV
            Environment.QA -> LogLevel.QA
            Environment.PROD -> LogLevel.PROD
        }
```

Was this an oversight, or am I misunderstanding the purpose of the logLevel property in LibrarySettings?

**Comment by jameskeith** [comment]
Hi @iankerr 
Thanks for raising this - I agree this looks like an oversight, so we'll get this addressed in a future release.

**Comment by iankerr** [comment]
Thanks. Related to this, collectDispatcherEnabled and tagManagementDispatcherEnabled seem to be in the same boat. They don't seem to be referenced outside their class and tests, and setting them to false don't actually disable the collect or tag management dispatchers.

**Comment by jameskeith** [comment]
fixes available in 1.6.0
